--- origsrc/ibus-chewing-1.3.9.2-Source/CMakeLists.txt	2010-11-18 20:41:24.000000000 -0600
+++ src/ibus-chewing-1.3.9.2-Source/CMakeLists.txt	2011-10-03 00:41:22.750476400 -0500
@@ -22,7 +22,7 @@ IF(CMAKE_VERSION)
 ENDIF(CMAKE_VERSION)
 
 INCLUDE(ManageVersion)
-LOAD_RELEASE_FILE("RELEASE-NOTES.txt")
+LOAD_RELEASE_FILE("${CMAKE_SOURCE_DIR}/RELEASE-NOTES.txt")
 INCLUDE(UseUninstall)
 
 INCLUDE(CompileEnv)
@@ -111,7 +111,7 @@ SET(PACK_SOURCE_IGNORE_FILES ${PACK_SOUR
     ${PRJ_IGNORE_FILES_COMMON} "/bin/" "\\\\.xml$" "\\\\.schemas")
 
 PACK_SOURCE(PACK_SOURCE_FILE_NAME "${RPM_BUILD_SOURCES}")
-PACK_RPM(PRJ_SRPM_FILE_NAME "${RPM_BUILD_SPECS}/project.spec.in" "${PACK_SOURCE_FILE_NAME}")
+PACK_RPM(PRJ_SRPM_FILE_NAME "${CMAKE_SOURCE_DIR}/SPECS/project.spec.in" "${PACK_SOURCE_FILE_NAME}")
 
 # If mock exists, then add mock related targets.
 FIND_PROGRAM(MOCK_CMD mock)
